docs/DeveloperGuide.md

```diff
@@ -346,42 +347,420 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
-`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+|`***`|New User|view instructions|read documentation on how to use the program|
+|`***`|User|add task|keep track of things to do|
+|`***`|User|delete a task|remove tasks I no longer want to keep track of|
+|`***`|User|complete task|update my progress|
+|`***`|User|edit task|correct or revise the task name and its details|
+|`***`|User|add tags to task|be able to group tasks under a category|
+|`***`|User|search for keyword|search for a task by keywords|
+|`***`|User|view incomplete tasks|keep track of what I still need to do|
+|`***`|User|view complete tasks|keep track of tasks I have completed|
+|`***`|User|view overdue tasks|know which tasks to prioritise finishing|
+|`***`|User|view task progress|keep track of my progress|
+|`***`|User|sync list with Google Calendar|view my todo list across different platforms|
```

Comment:
this is not part of the *must-haves* feature list. are you guys sure it should be `***` priority?